StyleCop.Analyzers/StyleCop.Analyzers/LayoutRules/SA1507CodeMustNotContainMultipleBlankLinesInARow.cs

```diff
@@ -54,16 +57,54 @@ public class SA1507CodeMustNotContainMultipleBlankLinesInARow : DiagnosticAnalyz
         /// <inheritdoc/>
         public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         {
-            get
-            {
-                return SupportedDiagnosticsValue;
-            }
+            get { return SupportedDiagnosticsValue; }
```

Comment:
@vweijsters @pdelvo I generally disagree with the exception, but not enough to require everything appear in the long format. However, I would prefer the pull request not change the layout of an existing block already in the preferred form.